From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jo Shields <joshield@microsoft.com>
Date: Mon, 23 Sep 2024 10:11:14 -0400
Subject: [PATCH] Rename SourceBuildUseMonoRuntime property, which is not SB
 specific

This is a coordinated cross-repo change, and might break the build until
all four PRs are merged. The other three are:

Backport: https://github.com/dotnet/source-build/issues/4165
---
 eng/DotNetBuild.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/DotNetBuild.props b/eng/DotNetBuild.props
index ce13d6ca5b0..316eea2522b 100644
--- a/eng/DotNetBuild.props
+++ b/eng/DotNetBuild.props
@@ -53,7 +53,7 @@
       <InnerBuildArgs>$(InnerBuildArgs) $(FlagParameterPrefix)verbosity $(LogVerbosity)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) $(FlagParameterPrefix)nodereuse $(ArcadeFalseBoolBuildArg)</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</InnerBuildArgs>
-      <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) $(FlagParameterPrefix)usemonoruntime</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(DotNetBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) $(FlagParameterPrefix)usemonoruntime</InnerBuildArgs>
       <!-- TODO: This parameter is only available on the Unix script. Intentional? -->
       <InnerBuildArgs Condition="'$(OS)' != 'Windows_NT'">$(InnerBuildArgs) --outputrid $(TargetRid)</InnerBuildArgs>
       <!-- PackageOS and ToolsOS control the rids of prebuilts consumed by the build.
